support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -327,12 +329,33 @@ EventsAndTriggers:
           # for visualization.
           CoordinatesFloatingPointType: Float
           FloatingPointTypes: [Float]
-  # Try to find common horizon at small separation
+# Trigger to find common horizon and gather volume data needed for ringdown.
   - Trigger:
-      SeparationLessThan:
-        Value: 2.5
+      And:
+        - SeparationLessThan:
+            Value: 2.38 # for q=1, spin=0, quasicircular
+        - Slabs:
+            EvenlySpaced:
+              Interval: 1
+              Offset: 0
```

Comment:
I believe this trigger (and the `And:`) are unnecessary because `SeparationLessThan` is checked every slab